presto-main/src/main/java/io/prestosql/server/protocol/QueryResultRows.java

```diff
@@ -150,10 +148,13 @@ public long getTotalRowsCount()
 
     private Optional<List<Object>> getRowValues()
     {
+        // types are present if data is present
+        List<ColumnAndType> columns = this.columns.get();
```

Comment:
```suggestion
        List<ColumnAndType> columns = this.columns.orElseThrow();
```

(`get()` triggers an IDE warning)